packages/react-scripts/scripts/eject.js

```diff
@@ -36,6 +37,31 @@ prompt(
     process.exit(1);
   }
 
+  // Make sure there are no dirty git status
+  function statusSync() {
+    try {
+      let stdout = execSync(`git status --porcelain`).toString();
+      let status = { dirty: 0, untracked: 0 };
```

Comment:
If you're also counting untracked files, why isn't that used in the dirty check below? If it shouldn't be used, why is it being counted?